webapp/channels/src/components/threading/global_threads/thread_list/thread_list.test.tsx

```diff
@@ -119,7 +119,7 @@ describe('components/threading/global_threads/thread_list', () => {
             <ThreadList {...props}/>,
         );
 
-        wrapper.find(Header).shallow().find({content: 'Mark all as read'}).find(Button).simulate('click');
+        wrapper.find(Header).shallow().find(Button).find({marginTop: true}).simulate('click');
```

Comment:
`find({marginTop: true}` instead of this can we have something more specific to the button instead of just margin top

Reply:
Updated, I used the `Id`